src/Carrier.php

```diff
@@ -98,16 +98,25 @@ public function __construct($csvCountryCode = null, $csvZoneToDeliveryMethod = n
      * sorted values to the RoyalMailMethod class to be
      * converted into objects.
      *
+     * The $ignore_package_value parameter allows for the
+     * value of the packages to be ignored in the calculation
+     * at the users discretion.
```

Comment:
I think it's worth updating this description of why this can be useful or in what situation this would be necessary and how it affects the return result.